### PR TITLE
added list.concat definition

### PR DESCRIPTION
**Pull request by Graur**
Closes: #24

### CODE REVIEW

**Comment by Graur** [comment]
@yegor256 please, have a look

File: paper.tex [hunk 1]
```diff
@@ -514,6 +514,8 @@ \subsection{List}
 
 The attribute \deff{reversed} is a new array with elements positioned in a reverse order.
 
+The attribute \deff{concat} is a new array combined current \ff{array} and passed array.
```
Comment (yegor256) [inline paper.tex:517]: @Graur I would say "a new array that concatenates the current array with the array provided as an argument".

**Comment by Graur** [comment]
@yegor256 Could you check again, please?

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor stop

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor stop

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**